dandiapi/api/services/embargo/__init__.py

```diff
@@ -0,0 +1,75 @@
+from django.conf import settings
+from django.db.models import QuerySet
+
+from dandiapi.api.copy import copy_object_multipart
+from dandiapi.api.models import Asset, AssetBlob, Dandiset, Upload, Version
+from dandiapi.api.tasks import unembargo_dandiset_task
+
+from .exceptions import AssetNotEmbargoed, DandisetNotEmbargoed
+
+
+def unembargo_asset(asset: Asset):
```

Comment:
I might `_` this for now, otherwise it's inviting people to call it without going through a method that does the proper permission checking. It also makes sense from the business logic perspective, we don't allow a user to unembargo an individual asset.